prog/github/repository_nexus.rb

```diff
@@ -73,9 +82,9 @@ def before_run
       nap 0
     end
 
-    check_queued_jobs
+    polling_interval = check_queued_jobs
```

Comment:
I really don't like a function named `check_queued_jobs` to return a polling interval. Maybe we can write it to `frame`?

Reply:
I will refactor it

Reply:
I changed it. Since `check_queued_jobs` isn't a separate label, it put the polling interval to `@polling_interval` variable.